src/lib.rs

```diff
@@ -63,9 +63,15 @@
 
 extern crate rand_core;
 
+#[cfg(feature = "std")]
+extern crate std;
```

Comment:
I guess this works... I think this would be more standard:
```
#![cfg_attr(not(feature = "std"), no_std)]
```

Reply:
If you do this, I think you can even get rid of:
``` 
#[cfg(feature = "std")]
use std::is_x86_feature_detected;
```

Reply:
I’ll integrate these myself (as I’ve already merged the PR locally)

Reply:
Doing things the other way causes a lot of pain wrt imports from core/std, so I’ll leave things as is.

Reply:
I intend to address the comments given by @jethrogb , @nagisa may I push my changes?

Reply:
> May I push my changes?

Sure, open a new PR with your changes. That being said `rdrand` implementation is full of 

```
use core::*
```

and similar references to core, which makes it fairly unpleasant to make a change like this and it would probably result in code that is in general worse than before IMO.